### PR TITLE
README: beta

### PR DESCRIPTION
**Pull request by xiang90**
/cc @hongchaodeng @philips

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,17 @@
 
 [![Build Status](https://jenkins-etcd-public.prod.coreos.systems/buildStatus/icon?job=etcd-operator-master)](https://jenkins-etcd-public.prod.coreos.systems/job/etcd-operator-master/)
 
-**Project status: *alpha*** Not all planned features are completed. The API, spec, status and other user facing objects are subject to change. We do not support backward-compatibility for the alpha releases.
+## Project status
+
+etcd operator is a **beta** software.
+
+Major planned features are completed. The etcd cluster spec, status are fairly stable. Any breaking changes of user facing objects or API will be pointed out in release notes.
```
Comment (hongchaodeng) [inline README.md]: remove "fairly".

Comment (hongchaodeng) [inline README.md]: After a second thought, I think we could better phrase that:

Major planned features are completed. Our API is beta. We don't intend to break any API unless we found out any mistake before stable. We will announce the changes to API in release notes.

**Comment by hongchaodeng** [comment]
@calebamiles Please also take a look

File: README.md (hunk 1 above)
Comment (xiang90) [inline README.md]: ok

**Comment by xiang90** [comment]
@hongchaodeng fixed. PTAL.

File: README.md [hunk 2]
```diff
@@ -2,7 +2,17 @@
 
 [![Build Status](https://jenkins-etcd-public.prod.coreos.systems/buildStatus/icon?job=etcd-operator-master)](https://jenkins-etcd-public.prod.coreos.systems/job/etcd-operator-master/)
 
-**Project status: *alpha*** Not all planned features are completed. The API, spec, status and other user facing objects are subject to change. We do not support backward-compatibility for the alpha releases.
+## Project status
+
+etcd operator is a **beta** software.
+
+Major planned features are completed. We don't plan to have any breaking API changes. However, we might fix API bugs before stable release that introduce backward incompatible behaviors. We will announce breaking changes in release notes.
```
Comment (hongchaodeng) [inline README.md]: However, before stable release, we might fix API bugs which could introduce ...

Comment (calebamiles) [inline README.md]: I would move project status to heading and expand the project introduction a bit 

```
### Project Status: beta

Major planned features have been completed and while no breaking API changes are currently planned, we reserve the right to address bugs and API changes in a backwards incompatible way before the project is declared stable. Any breaking changes introduced will be documented in release notes.

Currently user face etcd cluster objects are created as [Kubernetes Third Party Resources](https://kubernetes.io/docs/user-guide/thirdpartyresources/), however, taking advantage of [User Aggregated API Servers](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/aggregated-api-servers.md) to improve reliability, validation and versioning is planned. The use of Aggregated API should be minimally disruptive to existing users but may change what Kubernetes objects are created or how users deploy the etcd operator.

We expect to consider the etcd operator stable soon; backwards incompatible changes will not be made once the project reaches stability.
```

File: README.md [hunk 3]
```diff
@@ -2,7 +2,17 @@
...
+
+The current user facing etcd cluster objects are created as Kubernetes [TPR](https://kubernetes.io/docs/user-guide/thirdpartyresources/) items. There is a planned change to expose the object through [UAS](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/aggregated-api-servers.md) for better reliabity (validation, admission, versioning). It will not change spec or status. However, it might change the user facing object kind, and might affect how users deploy etcd operator. The overall impact should be minimal.
+
+We expect etcd operator to be stable soon. Backwards-incompatible changes will not be made after this project goes into stable stage.
+
+## Overview
 
 etcd operator manages etcd clusters atop [Kubernetes][k8s-home], automating their creation and administration:
```
Comment (calebamiles) [inline README.md]: I would maybe say something like

```
The etcd operator manages etcd clusters deployed to Kubernetes and automates tasks related to operating an etcd cluster which previously required deep knowledge of etcd
```

**Comment by hongchaodeng** [comment]
lgtm

**Comment by xiang90** [comment]
@calebamiles Good suggestions. I addressed the beta release related suggestions. I will cleanup other stuff you mentioned in another pr (i have a few other things that want to clean up along with yours for the readme).

**Pull request merged**